tests/test_profile_general.py

```diff
@@ -102,63 +103,214 @@
     "timestamps.csv",
 ]
 
-# logging function for threshold outliers set to false
-COUNTER_LOGGING = False
-METRIC_LOGGING = False
-
-# Absolute Difference < 2
-CONSISTENT_ABS_METRIC_INDICES = [
-    "2.1.8",
-    # "2.1.28",
-    "2.1.9",
-    "5.1.3",
-    "13.1.0",
-    "14.1.0",
-    "5.2.4",
-    "11.2.0",
-    "11.2.2",
-    "11.2.3",
-    "13.1.0",
-    "13.2.3",
-    "16.5.3",
-    "17.2.3",
-    "17.2.8",
-    "17.2.9",
-    "17.3.8",
-    "17.3.9",
-    "17.3.11",
-    "17.3.14",
-    "17.3.15",
-    "17.3.16",
-    "18.1.13",
-    "18.1.16",
-]
-# Percent Difference < 8
-CONSISTENT_REL_METRIC_INDICES = [
-    "2.1.26",
-    # "5.1.0",
-    # "5.1.1",
-    # "5.2.0",
-    # "5.2.1",
-    "5.2.3",
-    # "5.2.4",
-    "5.2.6",
-    "5.2.8",
-    # "6.1.0",
-    # "6.1.1",
-    "6.1.3" "6.1.6",
-    # "6.1.7",
-    # "6.2.0",
-    # "7.2.1",
-    # "7.2.3",
-    "7.2.4",
-    "7.2.7",
-    # "14.1.0",
-    "16.2.0",
-    # "16.3.14",
-    # "16.3.15",
-    "17.1.0",
+# Must not change relative difference is zero
+FIXED_METRICS = [
```

Comment:
What if we tweak this data structure to define whether we want it to use an absolute tolerance or relative tolerance for pass/fail?  Would prefer to start with coarser tolerances while we gain confidence across multiple SoCs and execute across multiple nodes.